tests/test_analysis_tools.py

```diff
@@ -108,16 +110,9 @@ def test_create_notebook(path_to_cif, vis_fxns):
     assert result == "Visualization Complete"
 
 
-@pytest.fixture
-def rmsd_functions(get_registry):
-    reg = get_registry("raw", True)
-    pdb_file = reg.get_mapped_path("ALA_123456")
-    rmsd_functions = RMSDFunctions(reg, pdb_file, "trajectory.dcd")
-    rmsd_functions.trajectory = "trajectory.dcd"
-    rmsd_functions.pdb_file = pdb_file
-    return rmsd_functions
-
+################ RMSD & PPI #################
```

Comment:
Instead, these should just be separate files

Reply:
moved to "test_analysis_rmsd_ppi.py" file